docs/transaction-api.md

```diff
@@ -219,10 +219,10 @@ The payment must be created with `quoteId`.
 - `COMPLETED`: The payment is either auto-completed once the received amount equals the expected `incomingAmount`, or it is completed manually via an API call.
 - `EXPIRED`: If the payment expires before it is completed then the state will move to `EXPIRED` and no further payments will be accepted.
 
-### `Account`
+### `PaymentPointer`
```

Comment:
Nit: should this be one word?

Reply:
I vote yes, since it's a type
https://github.com/interledger/rafiki/blob/main/docs/transaction-api.md#webhook-event